### PR TITLE
Add `hudochenkov/order`, Stylelint's Recommended and Standard for SCSS, and Prettier to the list of configs

### PR DESCRIPTION
**Pull request by Zwyx**
`hudochenkov/order` is a great config in my opinion, made by the creator of the `stylelint-order` plugin.

I also added Stylelint's Recommended and Standard configs for SCSS, as well as Prettier's configs.

> Which issue, if any, is this issue related to?

None, as it's a documentation fix.

> Is there anything in the PR that needs further explanation?

No, it's self-explanatory.

### CODE REVIEW

**Review by ybiquitous — COMMENTED** [review]
@Zwyx Thanks for the request. I agree with adding the config to the list, but one question. Why do you add `stylelint-config-hudochenkov/order` instead of `stylelint-config-hudochenkov`?

**Comment by Zwyx** [comment]
Thanks for your message @ybiquitous. Good question! I guess `stylelint-config-hudochenkov` is a config that "compete" with `stylelint-standard`. Whereas `stylelint-config-hudochenkov/order` really adds values that isn't part of the `stylelint-standard`. So I wanted to emphasise this.

But we can add `stylelint-config-hudochenkov` if you prefer, no worries at all. Let me know.

This made me thought that we could also add the recommended and standard configs for SCSS, as well as the Prettier one.

I also placed the Stylelint ones at the top of the list, and I added "_(extends the recommended one)_" to `stylelint-standard`, because it can be confusing to know which one of `standard` and `recommended` extends the other.

That became a lot of changes. Let me know if you don't like anything!

File: README.md [hunk 1]
```diff
@@ -15,11 +15,13 @@
 
 ## Configs
 
+- [stylelint-recommended](https://github.com/stylelint/stylelint-config-recommended) and [stylelint-recommended-scss](https://github.com/stylelint-scss/stylelint-config-recommended-scss) - The recommended shareable config for Stylelint.
+- [stylelint-standard](https://github.com/stylelint/stylelint-config-standard) and [stylelint-standard-scss](https://github.com/stylelint-scss/stylelint-config-standard-scss) - The standard shareable config for Stylelint (extends the recommended one).
+- [Prettier](https://github.com/prettier/stylelint-config-prettier) - Turns off all rules that are unnecessary when using Prettier.
```
Comment (ybiquitous) [inline README.md:22]: [suggest] How about adding also [`stylelint-config-prettier-scss`](https://github.com/prettier/stylelint-config-prettier-scss)?

**Review by ybiquitous — COMMENTED** [review]
I see. No problems to add `hudochenkov/order`.
And, also adding *official* configs sounds good. 👍🏼 

Can you please update the PR title for accuracy?

---

@hudochenkov If you have any opinions, please feel free to comment.

File: README.md (hunk 1 above)
Comment (Zwyx) [reply README.md:22]: Ah yes of course! Thanks for that. Title updated also.

**Review by ybiquitous — APPROVED** [review]
Thanks, nice update. LGTM 👍🏼

**Review by hudochenkov — APPROVED** [review]

**Pull request merged**